### ISSUE TITLE
training loss becomes nan when I train imagenet

### ISSUE THREAD

**Issue by localbetascreening** [hustvl/LightningDiT#10]
During training step around 194700, the training loss becomes nan. I used the demo of training epoch 64 to get FID 2.1. What's wrong?

**Comment by JingfengYao** [comment]
@localbetascreening  Maybe you should provide more details of your training.

**Comment by localbetascreening** [comment]
[log.txt](https://github.com/user-attachments/files/18394327/log.txt)
Please see the log.txt. As you can see, finally the model exports nan values.

**Comment by JingfengYao** [comment]
Thank you for your question.  

1. We’ve uploaded a [training log](https://github.com/hustvl/LightningDiT/blob/main/docs/log.txt) for reference.  
2. With a smaller global batch size in your experiment, consider lowering the learning rate and increasing the beta2 parameter of the AdamW optimizer. You may try the original DiT settings with a batch size of 256, a learning rate of 1e-4, and a beta2 of 0.999.
3. The loss at 180k seems reasonable—have you evaluated the generative performance of that checkpoint?

**Comment by localbetascreening** [comment]
> Thank you for your question.
> 
> 1. We’ve uploaded a [training log](https://github.com/hustvl/LightningDiT/blob/main/docs/log.txt) for reference.
> 2. With a smaller global batch size in your experiment, consider lowering the learning rate and increasing the beta2 parameter of the AdamW optimizer. You may try the original DiT settings with a batch size of 256, a learning rate of 1e-4, and a beta2 of 0.999.
> 3. The loss at 180k seems reasonable—have you evaluated the generative performance of that checkpoint?

Thanks for your reply. Yes, the result on 180k is Ok. I will try what you suggested! Thank you!
